### ISSUE TITLE
user-menu does not work when prefix set

### ISSUE THREAD

**Issue by ulno** [coderaiser/cloudcmd#244]
* **Version** (`cloudcmd -v`): 12.2,0
* **Node Version** `node -v`: 10 and 11
* **OS** (`uname -a` on Linux): raspbian, arch, and manjaro
* **Browser name/version**: firefox and chromium, latest versions
* **Used Command Line Parameters**: --user-menu 
* **Changed Config**:

I know, I should provide some updated documentation for user-menu (however, it's already pretty nice, just some minor changes that I shoudl deliver soon in a pull-request), instead I just came back to working with it and found a small bug:

If prefix is set (I set both prefix and prefixSocket) in the configuration (I have cloudcmd in my production version running under /cloudcmd prefix), the user-menu does not work. It never even shows up (only the changed button is visible).

If no prefix is used, it does work as described.

**Comment by coderaiser** [comment]
Thank you :). 

Looks like the problem in [client/modules/user-menu.js](https://github.com/coderaiser/cloudcmd/blob/master/client/modules/user-menu/index.js#L45).
`CloudCmd.prefix` should be used in a similar way as it used in [init](https://github.com/coderaiser/cloudcmd/blob/master/client/modules/user-menu/index.js#L28).

**Comment by ulno** [comment]
Ah, yes 12.2.0 - typo.

Is that a big issue to fix?

**Comment by coderaiser** [comment]
> Is that a big issue to fix?

No, I wrote how to fix it in a previous message :).

**Comment by ulno** [comment]
So just add the prefix to all loads?

**Comment by coderaiser** [comment]
Only to `fetch`

**Comment by ulno** [comment]
Changing the string of fetch [client/modules/user-menu.js](https://github.com/coderaiser/cloudcmd/blob/master/client/modules/user-menu/index.js#L45) to:
```js
const res = await fetch(`${CloudCmd.prefix}/api/v1/user-menu?dir=${dirPath}`);
```

This seems to do the trick - need to do some further testing, but need now first to deploy this to five machines (teaching this in five hours).


I will report here later if things worked.

Thanks for the last minute support.

**Comment by ulno** [comment]
So far it withstood the tests today. I created a PR but don't see it in the list (must have messed up something), so maybe you can just change this one line yourself?

**Comment by coderaiser** [comment]
Fixed with 8ac3211

**Comment by coderaiser** [comment]
Landed in `Cloud Commander` [v12.3.0](https://github.com/coderaiser/cloudcmd/releases/tag/v12.3.0) 🎉
